### PR TITLE
NT-0009 Add material-ui basic support. Use components from Material UI.

### PR DESCRIPTION
**Pull request by damian0o**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no"
+/>
```
Comment (mras2303) [inline client/public/index.html]: Closing tag in new line is necessary

File: client/src/components/Button/Button.stories.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 
-import { Button, ButtonA } from './Button';
+import { Button, ButtonA, ButtoMUI } from './Button';
```
Comment (mras2303) [inline client/src/components/Button/Button.stories.tsx]: Butto? Or Button?

File: client/src/components/Button/Button.stories.tsx [hunk 3]
```diff
@@ -27,4 +27,10 @@ storiesOf('Button', module)
 
 storiesOf('ButtonA', module).add('default', () => (
   <ButtonA onClick={action('clicked')}>Hello Button</ButtonA>
-));
\ No newline at end of file
+));
+
+storiesOf('ButtonMUi', module).add('default', () => (
+  <ButtoMUI>
+    Hello Material UI
```
Comment (mras2303) [inline client/src/components/Button/Button.stories.tsx]: Butto or Button?

File: client/src/components/Button/Button.stories.tsx [hunk 4]
```diff
@@ -27,4 +27,10 @@ storiesOf('Button', module)
 
 storiesOf('ButtonA', module).add('default', () => (
   <ButtonA onClick={action('clicked')}>Hello Button</ButtonA>
-));
\ No newline at end of file
+));
+
+storiesOf('ButtonMUi', module).add('default', () => (
+  <ButtoMUI>
```
Comment (mras2303) [inline client/src/components/Button/Button.stories.tsx]: Butto or Button?

File: client/src/components/Button/Button.stories.tsx [hunk 5]
```diff
@@ -27,4 +27,10 @@ storiesOf('Button', module)
 
 storiesOf('ButtonA', module).add('default', () => (
   <ButtonA onClick={action('clicked')}>Hello Button</ButtonA>
-));
\ No newline at end of file
+));
+
+storiesOf('ButtonMUi', module).add('default', () => (
```
Comment (mras2303) [inline client/src/components/Button/Button.stories.tsx]: I think UI should be written uppercase

File: client/src/components/Button/Button.tsx [hunk 6]
```diff
@@ -1,5 +1,7 @@
+import React from 'react';
 import styled, { css } from "styled-components";
 import posed from "react-pose";
+import ButtonMaterialUi from '@material-ui/core/Button';
```
Comment (mras2303) [inline client/src/components/Button/Button.tsx]: I think UI should be written uppercase

File: client/src/components/Button/Button.tsx [hunk 7]
```diff
@@ -58,3 +60,13 @@ export const ButtonA = posed(Button)({
     scale: 0.8
   }
 });
+
+interface ButtonMUIProp {
+  children: string
+}
+
+export const ButtoMUI = ({children}: ButtonMUIProp) => {
```
Comment (mras2303) [inline client/src/components/Button/Button.tsx]: Space between curly brackets and content

```({ children })```

File: client/src/components/MadeWithLove/MadeWithLove.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
```
Comment (mras2303) [inline client/src/components/MadeWithLove/MadeWithLove.stories.tsx]: unnecessary break between imports

File: client/src/components/MadeWithLove/MadeWithLove.stories.tsx [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+import { storiesOf } from '@storybook/react';
+
```
Comment (mras2303) [inline client/src/components/MadeWithLove/MadeWithLove.stories.tsx]: unnecessary break between imports

File: client/src/components/MadeWithLove/MadeWithLove.tsx [hunk 10]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
```
Comment (mras2303) [inline client/src/components/MadeWithLove/MadeWithLove.tsx]: unnecessary break between imports

File: client/src/components/MadeWithLove/index.ts [hunk 11]
```diff
@@ -0,0 +1 @@
+export { MadeWithLove as MadeWithLove } from './MadeWithLove';
```
Comment (mras2303) [inline client/src/components/MadeWithLove/index.ts]: Why did you use alias the same name as original?

Comment (damian0o) [reply client/src/components/MadeWithLove/index.ts]: I tried multiple things here forgot to remove

**Pull request merged**